reo/validators.py

```diff
@@ -1255,6 +1259,26 @@ def check_special_cases(self, object_name_path, template_values=None, real_value
                     # Use 2017 b/c it is most recent year that starts on a Sunday and all reference profiles start on
                     # Sunday
 
+                if real_values.get('outage_utility_name') != None and real_values.get('use_default_outage') == False:
+                    self.general_warnings.append("Overriding use_default_outage value, using the default outage duration")
+                    self.update_attribute_value(object_name_path, number, 'use_default_outage', True)
+                    real_values['use_default_outage'] = True
+                
+                if real_values.get('use_default_outage') == True:
```

Comment:
Can we remove the print below? In the message it would be nice to repeat the parsed utility name from the inputs. You might also consider linking to a place where the user can look up a list of valid utility names